config/transformObjectCatalog.py

```diff
@@ -0,0 +1,11 @@
+import os.path
+
+from lsst.utils import getPackageDir
+
+# Always produce these output bands in the Parquet coadd tables, no matter
+# what bands are in the input.
+# config.outputBands = ["g", "r", "i", "z", "y"]
+
+# Use the environment variable to prevent hardcoding of paths
+# into quantum graphs.
+config.functorFile = os.path.join('$OBS_DECAM_DIR', 'policy', 'Object.yaml')
```

Comment:
Please add a TODO comment with a link to some `drp_pipe`-flavored ticket that will move stuff out of `obs_decam/policy`, and do the same for `config/transformSourceTable.py` and anywhere else that depends on things in the policy directory.